### DISCUSSION TITLE
Error messaging when passthrough copy and templates attempt to write to the same place

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#4129] [Enhancement Queue]
We do have error messages for conflicting template output and conflicting passthrough copy targets, but do we have an error message when a permalink and passthrough copy both attempt to write to the same spot?

Inspired by #2761.

**Comment by zachleat** [comment]
Related: #2174 #562

**Comment by NiceOkay** [comment]
Hi @zachleat. I have error message when i use addPassthroughCopy whit the same output dir and option setServerPassthroughCopyBehavior('passthrough')

`
eleventyConfig.addPassthroughCopy({
  'src/assets': '/',
  'src/components/elements/card/assets': '/',
});
eleventyConfig.setServerPassthroughCopyBehavior('passthrough');
`

Error:
<img width="1234" alt="Снимок экрана 2023-06-19 в 20 57 44" src="https://github.com/11ty/eleventy/assets/44927185/24ac1cda-590e-4ce6-9202-595da7a49a62">
